EpikCord/__init__.py

```diff
@@ -1891,7 +1888,77 @@ def unload_section(self, section: Union[CommandsSection, EventsSection]):
 # class ClientGuildMember(Member):
 #     def __init__(self, client: Client,data: dict):
 #         super().__init__(data)
+class VoiceWebsocketClient:
+    def __init__(self, client:Client):
+        self.client = client
+        self.HELLO = 8
+        self.IDENTIFY = 0
+        self.HEARTBEAT = 3
+        self.sequence = None
+
+    async def connect(self, guild_id:str, channel_id:str, **kwargs):
+        self.guild_id = guild_id
+        await self.client.send_json({
+            "op": 4,
+            "d" : {
+                "guild_id": guild_id,
+                "channel_id": channel_id,
+                "self_mute": kwargs.get("self_mute", False),
+                "self_deaf" : kwargs.get("self_deaf", False)
+            }
+        })
+        #self.session_id = await self.client.ws.receive().json()
+        print(await (await self.client.ws.receive().json()))
```

Comment:
`.json()` is not awaitable, do `(await self.client.ws.receive()).json()` instead.